src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -272,14 +272,13 @@ private static int countUsedLines(DetailAST ast) {
         final Deque<DetailAST> nodes = new ArrayDeque<>();
         nodes.add(ast);
         final BitSet usedLines = new BitSet();
-        final int startLineNo = ast.getLineNo();
```

Comment:
This methods counts lines used in method. This logic returns the `cardinality` of the `BitSet` which is just how many lines were used. It doesn't matter what the starting line is as long as count the lines correctly (IE: Lines 17 and 18 is still 2 lines, just like lines 1 and 2).

With this `startLineNo` variable set to `0` for this mutation, all the other changes are basic math on their removal. `X - 0` is the same as just `X`. So the whole variable can just be removed.